packages/db/src/models/file.ts

```diff
@@ -21,6 +25,57 @@ class FilePgModel extends BasePgModel<PostgresFile, PostgresFileRecord> {
       .onConflict(['bucket', 'key'])
       .merge();
   }
+
+  /**
+   * Delete files from S3
+   *
+   * @param {Array<PostgresFileRecord>} files - A list of files with a bucket and key
+   * @returns {Promise}
+   * @private
+   */
+  async _deleteFilesFromS3(
```

Comment:
So this was part of the large refactor that's spread across a few commits. This logic now lives here:

https://github.com/nasa/cumulus/pull/2057/files#diff-e510f32e34f9005aa2bba79d3c157199f6f2509acac5c05a5cda62c755f8edf3R68-R91

We're don't have a direct replacement for this function because we're not deleting all of a granule's files in one transaction. We're deleting each file from PG and S3 individually.